Exiled.Events/Handlers/Server.cs

```diff
@@ -97,6 +102,8 @@ public class Server
         /// <param name="ev">The <see cref="ReportingCheaterEventArgs"/> instance.</param>
         public static void OnReportingCheater(ReportingCheaterEventArgs ev) => ReportingCheater.InvokeSafely(ev);
 
+        internal static void OnLocalReport(LocalReportEventArgs ev) => LocalReport.InvokeSafely(ev);
```

Comment:
And `OnLocalReporting`, also don't make `On` methods `internal`, so plugin devs can have their patches and invoke them as they want.